modules/social_features/social_core/tests/modules/social_core_test_entity_provider/social_core_test_entity_provider.info.yml

```diff
@@ -0,0 +1,9 @@
+name: "Social Core Entity Provider"
+type: module
+description: "A provider of an entity type."
+package: Testing
+core_version_requirement: ^10 || ^11
+php: 8.1
```

Comment:
I'm wondering if we should specify a php version here. I think it's used by Drupal's packager to put it into the composer.json but we don't use that. If we do have PHP requirements we should likely specify that in our `composer.json` directly instead :)

Reply:
I think we can remove this.

Reply:
removed, marking as resolved.